modules/CoreSampleGameplay/module.txt

```diff
@@ -4,7 +4,8 @@
     "displayName" : "Core Gameplay",
     "description" : "Minimal gameplay template. Little content but a few starting items.",
     "dependencies" : [
-        {"id": "Core", "minVersion": "2.0.0"}
+        {"id": "Core", "minVersion": "2.0.0"},
+        {"id" : "WildAnimalsGenome", "minVersion" : "0.2.0"}
```

Comment:
Do we really want the CoreSampleGameplay module to pull in WildAnimalsGenome? At least I think it's something that should be discussed. Or is it left over from debugging?

Reply:
Sorry, my mistake - leftovers from debugging, removed it.